### PR TITLE
Initial Endurance QARTOD values for the METBK

### PR DESCRIPTION
**Pull request by cwingard**
Initial QARTOD gross range and climatology test limits for the METBK sensor. Excluded sea_surface_conductivity from the climatology tests in keeping with the spirit of the QARTOD manual for CTDs, though I did calculate the climatology and the tables are included. Commented out the barometric_pressure climatology as the variance explained values are very low. Left them as placeholders if we decide to include them. Will re-run the code to create the limits once review annotations are pushed and will update any sensor that is significantly different.

### CODE REVIEW

**Review by crisien — CHANGES_REQUESTED** [review]
The values all look OK to me except for the shortwave irr for CE02, CE04 and CE07 where there are positive lower bound values rather than zero. CE09 shortwave irr looks OK.

**Comment by reedan88** [comment]
Looks generally good to me. I have a question about the gross range sensor range you used for the wind speeds. The instrument specs say wind speed limit of 0 to 65 m/s. My reading of the method of operation for the WindObserver II is that the range we use should be (-65, 65) because it can measure up to 65 m/s in any one direction since ws = sqrt(u2 + v2). Or if want to use the max wind speed at 45 degree angles to N & E the range should probably be +/- 45 m/s.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 1]
```diff
@@ -1,25 +1,89 @@
...
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'met_salsurf'},"{'qartod': {'gross_range_test': {'suspect_span': [26.02579, 33.54789], 'fail_span': [0.0, 42.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'eastward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-9.92, 9.77], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:11]: Is the gross range sensor range here correct? The manual for the WindObserver II on the ASIMET say 0-65 m/s, which using minimum values for u and v would be 45 m/s for both vectors.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 2]
```diff
@@ -1,25 +1,89 @@
...
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'northward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-10.4, 15.45], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:12]: Same question on the sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 3]
```diff
@@ -1,25 +1,89 @@
...
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'met_salsurf'},"{'qartod': {'gross_range_test': {'suspect_span': [26.02579, 33.54789], 'fail_span': [0.0, 42.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'eastward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-9.92, 9.77], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:22]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 4]
```diff
@@ -1,25 +1,89 @@
...
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'northward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-10.4, 15.45], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:23]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 5]
```diff
@@ -1,25 +1,89 @@
...
+CE04OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'met_salsurf'},"{'qartod': {'gross_range_test': {'suspect_span': [26.66549, 32.99193], 'fail_span': [0.0, 42.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE04OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'eastward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-8.62738, 10.58], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:33]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 6]
```diff
@@ -1,25 +1,89 @@
...
+CE04OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'northward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-10.65, 15.615], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:34]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 7]
```diff
@@ -1,25 +1,89 @@
...
+CE04OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'met_salsurf'},"{'qartod': {'gross_range_test': {'suspect_span': [26.66549, 32.99193], 'fail_span': [0.0, 42.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE04OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'eastward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-8.62738, 10.58], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:44]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 8]
```diff
@@ -1,25 +1,89 @@
...
+CE04OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'northward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-10.65, 15.615], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:45]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 9]
```diff
@@ -1,25 +1,89 @@
...
+CE07SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'met_salsurf'},"{'qartod': {'gross_range_test': {'suspect_span': [22.34751, 33.16531], 'fail_span': [0.0, 42.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE07SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'eastward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-11.66, 10.34], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:55]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 10]
```diff
@@ -1,25 +1,89 @@
...
+CE07SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'northward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-7.31, 12.6299], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:56]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 11]
```diff
@@ -1,25 +1,89 @@
...
+CE07SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'met_salsurf'},"{'qartod': {'gross_range_test': {'suspect_span': [22.34751, 33.16531], 'fail_span': [0.0, 42.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE07SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'eastward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-11.66, 10.34], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:66]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 12]
```diff
@@ -1,25 +1,89 @@
...
+CE07SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'northward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-7.31, 12.6299], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:67]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 13]
```diff
@@ -1,25 +1,89 @@
...
+CE09OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'met_salsurf'},"{'qartod': {'gross_range_test': {'suspect_span': [26.30023, 32.70528], 'fail_span': [0.0, 42.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE09OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'eastward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-11.93, 11.51853], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:77]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 14]
```diff
@@ -1,25 +1,89 @@
...
+CE09OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'northward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-8.18426, 13.98], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:78]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 15]
```diff
@@ -1,25 +1,89 @@
...
+CE09OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'met_salsurf'},"{'qartod': {'gross_range_test': {'suspect_span': [26.30023, 32.70528], 'fail_span': [0.0, 42.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE09OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'eastward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-11.93, 11.51853], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:88]: Same question on sensor range.

File: qartod/metbk/metbk_qartod_gross_range_test_values.csv [hunk 16]
```diff
@@ -1,25 +1,89 @@
...
+CE09OSSM,SBD11,06-METBKA000,metbk_a_dcl_instrument_recovered,{'inp': 'northward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-8.18426, 13.98], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```
Comment (reedan88) [inline qartod/metbk/metbk_qartod_gross_range_test_values.csv:89]: Same question on sensor range.

**Review by reedan88 — APPROVED** [review]
Looks good except for the question on the sensor range for the eastward_wind_velocity and northward_wind_velocity.

**Review by reedan88 — APPROVED** [review]

**Review by bkuebel — APPROVED** [review]

**Pull request closed without merging**